app/models/facility.rb

```diff
@@ -18,7 +18,8 @@ class Facility < ApplicationRecord
   has_many :phone_number_authentications, foreign_key: "registration_facility_id"
   has_many :users, through: :phone_number_authentications
   has_and_belongs_to_many :teleconsultation_medical_officers,
-    class_name: "TeleconsultationMedicalOfficer",
+    -> { distinct },
+    class_name: "User",
```

Comment:
Turns out doing `has_many` with `class_name: TeleconsultationMedicalOfficer` does not work out for us as expected. The `default_scope` on `TeleconsultationMedicalOfficer` means there is no way to check if a user is already added as an MO to a facility, since the results are already scoped. This was also somewhat incorrect, semantically. 

Still keeping the `TeleconsultationMedicalOfficer` class around for other semantic goodness.